Fabric.Identity.API/Persistence/SqlServer/Stores/SqlServerUserStore.cs

```diff
@@ -66,9 +58,8 @@ public SqlServerUserStore(IIdentityDbContext identityDbContext)
         public async Task<User> AddUserAsync(User user)
         {
             var userEntity = user.ToEntity();
-
+            userEntity.Claims = user.Claims.Select(c => new UserClaim {Type = c.Type, Value = c.Value}).ToList();
```

Comment:
you could move this line into the ToEntity extension method that does not take an existing entity as a parameter.

Reply:
Good suggestions, I done that for both this comment and the one below.